pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go

```diff
@@ -27,5 +31,76 @@ func (p plugin) MatchedPolicies(dataplane *core_mesh.DataplaneResource, resource
 
 func (p plugin) Apply(rs *core_xds.ResourceSet, ctx xds_context.Context, proxy *core_xds.Proxy) error {
 	log.V(1).Info("apply is not implemented")
+	policies, ok := proxy.Policies.Dynamic[api.MeshMetricType]
+	if !ok || len(policies.SingleItemRules.Rules) == 0 {
+		return nil
+	}
+
+	conf := policies.SingleItemRules.Rules[0].Conf.(api.Conf)
```

Comment:
don't we need to call "Compute" or something like this on it?

Reply:
since it is `SingleItemRule` we don't need to run `Compute` because there will be only one rule

Reply:
xrel: https://github.com/kumahq/kuma/issues/8681